sim/core/mana.go

```diff
@@ -22,6 +22,7 @@ type manaBar struct {
 	manaNotCastingMetrics *ResourceMetrics
 	JowManaMetrics        *ResourceMetrics
 	VtManaMetrics         *ResourceMetrics
+	JowiseManaMetrics     *ResourceMetrics
```

Comment:
is this different than `JowManaMetrics` two lines above?

Reply:
Yes it's 
https://wotlkdb.com/?spell=31878 (Judgement of the Wise)
instead of 
https://wotlkdb.com/?spell=53408 (Judgement of Wisdom)